traffic_portal/app/src/modules/private/cdns/config/ConfigController.js

```diff
@@ -56,33 +56,53 @@ var ConfigController = function(cdn, currentSnapshot, newSnapshot, $scope, $stat
 		if (oldJSON) {
 			var diff = JsDiff.diffJson(oldJSON, newJSON);
 			diff.forEach(function(part){
-				var partChanged = part.added || part.removed;
-				if (!partChanged && part.count > (context * 2)) {
-					var partArr = part.value.split("\n"),
-						newArr = [];
-					_.each(partArr, function(element, index) {
-						if (index < context || index > partArr.length - context) {
-							newArr.push(element);
-						}
-					});
-					newArr.splice(context, 0, "\n*****************\n*   TRUNCATED   *\n*****************\n");
```

Comment:
the reason i added the TRUNCATED part is because without it, this view will show the ENTIRE crconfig.json file which is massive (i think 9 mb or something) and the browser will puke and lock up. 

TRUNCATED basically truncates the non-changed parts rather than their entire contents.

Reply:
Yeah, I'll put that back in once it's working. Do you think that could be why nothing's showing up?